api_devauth.go

```diff
@@ -235,8 +235,7 @@ func (d *DevAuthHandler) UpdateDeviceStatusHandler(w rest.ResponseWriter, r *res
 		return
 	}
 
-	w.Header().Add("Location", "../"+devid)
-	w.WriteHeader(http.StatusSeeOther)
+	w.WriteHeader(http.StatusOK)
```

Comment:
Won't be able to change this later, once released will stay with us for long time. It's nice to keep it consistent, makes it more polished.